sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample5_AnalyzeConversation_ConversationPii.cs

```diff
@@ -0,0 +1,189 @@
+                }
+            }
+            #endregion
+
+            Assert.NotZero(entitiesDetected.Count);
+            Assert.That(analyzeConversationOperation.GetRawResponse().Status, Is.EqualTo(200));
+        }
+
+        [AsyncOnly]
+        [RecordedTest]
+        [ServiceVersion(Min = ConversationAnalysisClientOptions.ServiceVersion.V2024_05_01)]
+        public async Task AnalyzeConversationAsync_ConversationPii()
+        {
+            ConversationAnalysisClient client = Client;
+            List<NamedEntity> entitiesDetected = new();
+
+            AnalyzeConversationOperationInput data = new AnalyzeConversationOperationInput(
+                new MultiLanguageConversationInput(
+                    new List<ConversationInput>
+                    {
+                        new TextConversation("1", "en", new List<TextConversationItem>()
+                        {
+                            new TextConversationItem(id: "1", participantId: "Agent_1", text: "Can you provide you name?"),
+                            new TextConversationItem(id: "2", participantId: "Customer_1", text: "Hi, my name is John Doe."),
+                            new TextConversationItem(id : "3", participantId : "Agent_1", text : "Thank you John, that has been updated in our system.")
+                        })
+                    }),
+                    new List<AnalyzeConversationOperationAction>
+                    {
+                        new PiiOperationAction()
+                        {
+                            ActionContent = new ConversationPiiActionContent(),
+                            Name = "Conversation PII",
+                        }
+                    });
+
+            #region Snippet:AnalyzeConversationAsync_ConversationPii
+
+            Response<AnalyzeConversationOperationState> analyzeConversationOperation = await client.AnalyzeConversationOperationAsync(data);
+            #endregion
+
+            AnalyzeConversationOperationState operationState = analyzeConversationOperation.Value;
+
+            foreach (AnalyzeConversationOperationResult operationResult in operationState.Actions.Items)
+            {
+                Console.WriteLine($"Operation action name: {operationResult.Name}");
+
+                if (operationResult is ConversationPiiOperationResult piiOperationResult)
+                {
+                    foreach (ConversationalPiiResultWithResultBase conversation in piiOperationResult.Results.Conversations)
+                    {
+                        Console.WriteLine($"Conversation: #{conversation.Id}");
+                        Console.WriteLine("Detected Entities:");
+                        foreach (ConversationPiiItemResult item in conversation.ConversationItems)
+                        {
+                            foreach (NamedEntity entity in item.Entities)
+                            {
+                                Console.WriteLine($"Category: {entity.Category}");
```

Comment:
Could not find it in the PR for now so making sure to ask here so we make sure to look: 
Do we make sure that Name is available as category in convpii for 2022 api and Person from 2023?

Reply:
Discussed offline. No longer needed.